packages/mobile/src/identity/verification.ts

```diff
@@ -866,6 +873,79 @@ async function postToAttestationService(
   }
 }
 
+// Report to analytics reveal status from validator
+export function* reportRevealStatusSaga({
+  attestationServiceUrl,
+  e164Number,
+  account,
+  issuer,
+  pepper,
+}: ReportRevealStatus) {
+  if (shouldUseProxy()) {
+    // TODO We need to set proxy for this
+  } else {
+    const contractKit = yield call(getContractKit)
+    const attestationsWrapper: AttestationsWrapper = yield call([
+      contractKit.contracts,
+      contractKit.contracts.getAttestations,
+    ])
+    Logger.debug(
+      `${TAG}@reportAttestationRevealStatus`,
+      `Start for service url ${attestationServiceUrl}`
+    )
+    const response = yield call(
+      attestationsWrapper.getRevealStatus,
+      e164Number,
+      account,
+      issuer,
+      attestationServiceUrl,
+      pepper
+    )
+    const body = yield call(response.json.bind(response))
+    if (response.ok) {
+      Logger.debug(
+        `${TAG}@reportAttestationRevealStatus`,
+        `Successful for service url ${attestationServiceUrl}`
+      )
+      ValoraAnalytics.track(VerificationEvents.verification_reveal_attestation_status, body)
+    } else {
+      Logger.debug(
```

Comment:
any way to parse the response to ignore if the `get_attestation` endpoint doesn't exist for the service issuer -- to make this backwards compatible for older versions of attestation services that don't have this endpoint yet rather than logging it as a failure?